components/user_settings/display/user_settings_theme/user_settings_theme.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import SettingItemMin from '../../../setting_item_min';
+
+import Expanded from './expanded';
+
+const UserSettingsTheme: React.FC<Props> = ({selected, updateSection, setRequireConfirm, setEnforceFocus}: Props) => {
+    return selected ? (
+        <Expanded
+            updateSection={updateSection}
+            setRequireConfirm={setRequireConfirm}
+            setEnforceFocus={setEnforceFocus}
+        />
+    ) : (
+        <SettingItemMin
+            title={
+                <FormattedMessage
+                    id='user.settings.display.theme.title'
+                    defaultMessage='Theme'
+                />
+            }
+            describe={
+                <FormattedMessage
+                    id='user.settings.display.theme.describe'
+                    defaultMessage='Open to manage your theme'
+                />
+            }
+            section={'theme'}
+            updateSection={updateSection}
+        />
+    );
+};
+
+type Props = {
```

Comment:
I'm moving them to top - it'll be consistent with rest of codebase.
Initially I added to bottom because I'm used to Uncle Bob's (Clean Code) Stepdown Rule https://gist.github.com/delbetu/3fed29d1f56088938b019bc3b99bba72#stepdown-rule